File: 4268-116-1768-merge-strings-alternately/4268-116-1768-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int len = min(word1.size(), word2.size());
        string merger;
        int i = 0;
        for (; i<len; i++){
            merger += word1[i];
            merger += word2[i];
        }
        for (; i<word1.size(); i++)
            merger += word1[i];
        for (; i<word2.size(); i++)
            merger += word2[i];
        return merger;
    }
};

File: 4268-116-1768-merge-strings-alternately/4268-116-1768-merge-strings-alternately.java
class Solution {
    public String mergeAlternately(String word1, String word2) {
        int len = Math.min(word1.length(), word2.length());
        String merger = "";
        int i = 0;
        for (; i<len; i++){
            merger += word1.charAt(i);
            merger += word2.charAt(i);
        }
        for (; i<word1.length(); i++)
            merger += word1.charAt(i);
        for (; i<word2.length(); i++)
            merger += word2.charAt(i);
        return merger;
    }
}